eng/common-tests/spelling/check-spelling-in-changed-files.tests.ps1

```diff
@@ -8,6 +8,16 @@ Describe 'Spell checking' -Tag 'UnitTest' {
         Push-Location $workingDirectory | Out-Null
         git init
 
+        # Set up git user information for this specific repo if no user
+        # information is set globally. Windows DevOps machines do not have a
+        # global git identity set.
+        if (!(git config --get user.email)) {
+            git config user.email "test@contoso.com"
```

Comment:
I realize this is just for testing but we should consider just passing these in with the commit commands to avoid any contamination of the global git state.

Reply:
/# mkdir test1
/# cd test1
/test1# git init
...
Initialized empty Git repository in /test1/.git/
/test1# git config user.email
/test1# git config user.email "test@contoso.com"
/test1# git config user.email
test@contoso.com

# SAME MACHINE, DIFFERENT REPO

/test1# mkdir ../test2
/test1# cd ../test2
/test2# git init
...
Initialized empty Git repository in /test2/.git/
/test2# git config user.email
/test2#

Reply:
These tests create their own temp folder for testing outside of the scope of the repository. Changes don't affect the `azure-sdk-tools` repo.